frontend/src/lib/lemon-ui/LemonSelectMultiple/LemonSelectMultiple.stories.tsx

```diff
@@ -80,7 +78,28 @@ Loading.parameters = {
 
 export const NoOptions: Story = Template.bind({})
 NoOptions.args = {
-    mode: 'multiple-custom',
+    mode: 'multiple',
+    allowCustomValues: true,
     placeholder: 'No options...',
     options: [],
 }
+
+export const SingleOption: Story = Template.bind({})
+SingleOption.args = {
+    mode: 'single',
+    placeholder: 'Only one option allowed',
+}
+
+export const SingleOptionWithCustom: Story = Template.bind({})
+SingleOption.args = {
+    mode: 'single',
+    allowCustomValues: true,
```

Comment:
Ah, just tested this component and it seems like I _can't_ add custom values here.

<img width="796" alt="Screenshot 2024-03-15 at 15 09 38" src="https://github.com/PostHog/posthog/assets/6685876/3fecc08e-9dca-402c-9322-cf7d05a2c49d">

Looks like the boolean just needs to be inverted somewhere